app/models/concerns/hud_reports/length_of_stays.rb

```diff
@@ -85,24 +85,25 @@ module HudReports::LengthOfStays
     end
 
     private def date_to_street(enrollment, reporting_age, hoh_enrollment)
-      return enrollment.enrollment.DateToStreetESSH unless hoh_enrollment&.first_date_in_program == enrollment.first_date_in_program
-      return hoh_enrollment&.enrollment&.DateToStreetESSH if reporting_age.present? && reporting_age <= 17
+      return hoh_enrollment&.enrollment&.DateToStreetESSH if reporting_age.present? && reporting_age <= 17 && hoh_enrollment&.first_date_in_program == enrollment.first_date_in_program
 
       enrollment.enrollment.DateToStreetESSH
     end
 
     private def approximate_time_to_move_in(enrollment, reporting_age, hoh_enrollment)
       # PSH/RRH w/ move in date
       # OR project type 7 (other) with Funder 35 (Pay for Success)
-      move_in_date = if enrollment.project_type.in?(HudUtility2024.residential_project_type_numbers_by_code[:ph]) || enrollment.project.pay_for_success?
+      valid_ph_projects = HudUtility2024.permanent_housing_project_types - [HudUtility2024.project_type_brief('PH - OPH', true)]
+      other_pay_for_success = enrollment.project_type.in?([HudUtility2024.project_type_brief('Other', true)]) && enrollment.project.pay_for_success?
+      move_in_date = if enrollment.project_type.in?(valid_ph_projects) || other_pay_for_success
```

Comment:
Maybe this to clarify a bit?

```suggestion
      valid_ph_project_types = HudUtility2024.permanent_housing_project_types - [HudUtility2024.project_type_brief('PH - OPH', true)]
      valid_ph_enrollment = enrollment.project_type.in?(valid_ph_project_types)
      other_pay_for_success = enrollment.project_type.in?([HudUtility2024.project_type_brief('Other', true)]) && enrollment.project.pay_for_success?
      move_in_date = if  valid_ph_enrollment || other_pay_for_success
```